.github/workflows-src/steps.lib.yml

```diff
@@ -84,10 +84,18 @@ run: |
 #@ def steps_inttest():
 name: "Integration Tests"
 shell: bash
-run: go test -timeout 20m ./test/integration -v
+run: SHELL='' go test -timeout 20m -tags 'state_int_tests' ./test/integration -v
 env:
   INTEGRATION_TEST_USERNAME: ${{ secrets.INTEGRATION_TEST_USERNAME }}
   INTEGRATION_TEST_PASSWORD: ${{ secrets.INTEGRATION_TEST_PASSWORD }}
+  PLATFORM_API_TOKEN: ${{ secrets.PLATFORM_API_TOKEN }}
```

Comment:
@Naatan The secret still needs to be defined in Github.  But I do not have the permissions for that...

Reply:
Done